src/form/input-wrapper.js

```diff
@@ -101,39 +102,10 @@ export class InputWrapper extends Component {
     return render({
       value: this.formatNumber(),
       update: updateFormModel.bind(null, name, type || 'text'),
```

Comment:
in the other PR you had changed the implementation of `update` to `onChange`... however suggesting we keep update based on my next comment..

Reply:
additionally, now that you've added 'text' as a defaultProp above, the `|| 'text'` is no longer necessary